Services/Services.Domain/Enums/DoctorStatus.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Services.Domain.Enums
+{
+    public enum DoctorStatus
+    {
+        AtWork,
```

Comment:
There is Microsoft recommendation to always add a default value to enums like None

Reply:
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/enum

Reply:
Fixed